### PR TITLE
Wales update

### PR DESCRIPTION
**Pull request by d70-t**
This update covers the new `cloud_mask` of the WALES 2.1 dataset.

### CODE REVIEW

File: how_to_eurec4a/wales.md [hunk 1]
```diff
@@ -58,76 +59,121 @@ ds_cloud_sel = ds_cloud.sel(time=slice(datetime.datetime(2020, 2, 5, 13, 6, 30),
...
 ```
 
-We can use a time averaging window to derive a cloud fraction from the `cloud_mask` variable and see how it varies over the course of the flight. We further combine the two cloud flags `probably_cloudy` and `most_likely_cloudy` to a binary cloud mask.
+As it turns out, in this section of the flight, the instrument is very certain about the cloudiness.
+Yet, this missing value story was a bit complicated, so let's check if we would get something different if we would do it the naive way:
 
 ```{code-cell} ipython3
-ds_cloud["cloud_mask_binary"] = (ds_cloud.cloud_mask==1) | (ds_cloud.cloud_mask==2)
+cf_wrong = (ds_cloud.cloud_mask == cloudy_flags).any("flags").mean().values
+print("wrong cloud fraction: {:.2f}%".format(cf_wrong * 100))
```
Comment (tmieslinger) [inline how_to_eurec4a/wales.md:158]: To stay consistent with the formatting within this notebook I'd suggest:
```suggestion
print(f"wrong cloud fraction: {cf_wrong * 100:.2f} %")
```

File: how_to_eurec4a/wales.md [hunk 2]
```diff
@@ -58,76 +59,121 @@ ds_cloud_sel = ds_cloud.sel(time=slice(datetime.datetime(2020, 2, 5, 13, 6, 30),
...
+    averaged_cloud_fraction = min_cloud_binary_mask.resample(time=f"{t}min", loffset=f"{t/2}min").mean()
+    averaged_cloud_fraction.plot(lw=ind + 1, label=f"{t} min")
 
 ax.set_ylim(0, 1)
 ax.set_ylabel("Cloud fraction")
 ax.set_xlabel("date: MM-DD HH")
 ax.spines['right'].set_visible(False)
 ax.spines['top'].set_visible(False)
-ax.legend(title="averaging period", bbox_to_anchor=(1,1), loc="upper left")
+ax.legend(title="averaging period", bbox_to_anchor=(1,1), loc="upper left");
```
Comment (tmieslinger) [inline how_to_eurec4a/wales.md:178]:
```suggestion
ax.legend(title="averaging period", bbox_to_anchor=(1,1), loc="upper left")
None
```

**Review by tmieslinger — APPROVED** [review]
Thanks for updating the notebooks and adjusting them to the new data set!
Concerning the WALES notebook I was wondering whether the missing value handling "tutorial" deserves a better place where people will find it. So the corrections to the derivation of the cloud fraction are important and shall stay! But we might want to move the descriptive part including the `note` to a new place, for example the upcoming `cloud mask` notebook. However, I'd be fine with having the information here as long as the other notebook is still under construction. Or you can move it right away and add it to the pull request #43 Whatever you like better :-)

**Comment by d70-t** [comment]
I'd prefer to keep the explanation until the cloud mask notebook is merged. This way, the information is probably available earlier and could help to prevent nasty errors.

**Pull request merged**